Resources/Prototypes/_NF/Entities/Objects/Fun/spaceblade.yml

```diff
@@ -0,0 +1,249 @@
+- type: entity
+  id: SpaceBlade
+  parent: AirTank
+  name: space blade
+  description: Let it rip
```

Comment:
I hate fun - could this description be _more_ legally distinct?

Reply:
Fixed